pkg/extension/opampextension/README.md

```diff
@@ -69,16 +69,16 @@ Here is a list of valid values for the OpAMP `endpoint** configuration option:
 
 **Note:** These endpoints are not yet available.
 
-|  Deployment   | API base URL                                |
-|:-------------:|---------------------------------------------|
-| default/`US1` | `https://opamp-collectors.sumologic.com`     |
-|     `US2`     | `https://opamp-collectors.us2.sumologic.com` |
-|     `AU`      | `https://opamp-collectors.au.sumologic.com`  |
-|     `DE`      | `https://opamp-collectors.de.sumologic.com`  |
-|     `EU`      | `https://opamp-collectors.eu.sumologic.com`  |
-|     `JP`      | `https://opamp-collectors.jp.sumologic.com`  |
-|     `CA`      | `https://opamp-collectors.ca.sumologic.com`  |
-|     `IN`      | `https://opamp-collectors.in.sumologic.com`  |
+|  Deployment   | API base URL                                        |
```

Comment:
API base URL is a different concept than websocket URL. There should be a different table for that.